src/containers/booksList.js

```diff
@@ -1,21 +1,21 @@
-/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/prop-types */
```

Comment:
Add `propTypes` validation, don't disable the rule.

Reply:
DONE